system/plutono/templates/plutono-secrets.yaml

```
@@ -12,4 +12,5 @@ data:
   elk_password: {{ required ".Values.authentication.elk_password missing" .Values.authentication.elk_password | b64enc | quote }}
   opensearch_password: {{ required ".Values.authentication.opensearch_password missing" .Values.authentication.opensearch_password | b64enc | quote }}
   metisdb_password: {{ required ".Values.authentication.metisdb_password missing" .Values.authentication.metisdb_password | b64enc | quote }}
+  locust_timescaledb_password: {{ required ".Values.authentication.locust_timescaledb_password missing" .Values.authentication.locust_timescaledb_password | b64enc | quote }}
```

Required means the value is set in all Plutono regions? I guess that won't work since you only need it in `eu-nl-1`.

---

We can consider default empty string.
```suggestion
  locust_timescaledb_password: {{ default "" .Values.authentication.locust_timescaledb_password | b64enc | quote }}
```